drivers/hmis/app/graphql/schema.graphql

```diff
@@ -296,9 +296,9 @@ type AssessmentEligibility {
 }
 
 input AssessmentFilterOptions {
+  assessmentName: [String!]
```

Comment:
What is the best way to roll out this breaking API change? Maybe we need to take a phased approach, adding the new `assessmentName` and then removing the old `type` in a future release? From your comment [here](https://github.com/greenriver/hmis-frontend/pull/744#discussion_r1578513276) it sounds like you weren't too worried about the breaking change and we can just push this as-is?

Reply:
IMO we can live with this because there are no pages that use this filter by default. The user has to go out of their way to apply the filter. So the breaking change would only cause an issue if a user with a stale frontend during a deploy happens to apply the filter. I think we can just let that happen and proceed with the change.